app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -498,6 +498,9 @@ private void updateNearbyNotification(@Nullable NearbyController.NearbyPlacesInf
             // Means that no close nearby place is found
             nearbyNotificationCardView.setVisibility(View.GONE);
         }
+        if(mediaDetailPagerFragment!=null && !contributionsListFragment.isVisible()) {
```

Comment:
Would you mind adding a comment `// Prevent Nearby banner from appearing in Media Details, fixing bug https://github.com/commons-app/apps-android-commons/issues/4731`?

Also, would you mind following the whitespace conventions at https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html? Hint: spaces after `if` and around `!=`.

Thanks a lot! :-)